sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageAggregator.java

```diff
@@ -60,11 +60,11 @@ public CoverageAggregator(CoverageConfiguration coverageConf,
 
   @VisibleForTesting
   CoverageAggregator(CoverageConfiguration coverageConf, Configuration configuration,
```

Comment:
pedantic: we can move the configuration parameter on the next line to have them all aligned